app/assets/stylesheets/application.scss

```
@@ -8,18 +8,34 @@
   text-align: center;
   margin-top: 5px;
 }
+
+.hamburger-menu{
+ margin-left: 95%;
+ margin-top: -10%;
+ }
```

```suggestion
.hamburger-menu {
   margin-left: 95%;
   margin-top: -10%;
 }
```
code styling consistency 👀

---

Got it